force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,92 @@
+<template>
+    <lightning-card title="PD1 Calculator" class="slds-grid slds-grid_align-center">
+        <!-- Process -->
+        <template lwc:if={isFieldValueTooHigh}>
+            <lightning-button variant="brand" label="Calculate" slot="actions" disabled class="slds-var-m-left_x-small"></lightning-button>
+        </template>
+        <template lwc:else>
+            <lightning-button variant="brand" label="Calculate" slot="actions" onclick={calculateScore}></lightning-button>
```

Comment:
Try removing the calculateScore button and whenever the values are changed it automatically calculates.